src/pages/workspace/members/WorkspaceInviteMessageComponent.tsx

```diff
@@ -66,6 +66,30 @@ function WorkspaceInviteMessageComponent({
 }: WorkspaceInviteMessageComponentProps) {
     const styles = useThemeStyles();
     const {translate, formatPhoneNumber} = useLocalize();
+    const policyName = policy?.name;
+
+    const isExpensesFromRoute = useMemo(() => {
```

Comment:
```suggestion
    const isWorkflowApprovalExpensesFromRoute = useMemo(() => {
```

Should we name it more clearer?